src/Kovan-DssSpell.t.sol

```diff
@@ -179,16 +148,18 @@ contract DssSpellTest is DSTest, DSMath {
         // Test for all system configuration changes
         //
         afterSpell = SystemValues({
-            dsr_rate:     0,               // In basis points
-            vat_Line:     1227 * MILLION,  // In whole Dai units
-            pause_delay:  60,              // In seconds
-            vow_wait:     3600,            // In seconds
-            vow_dump:     2,               // In whole Dai units
-            vow_sump:     50,              // In whole Dai units
-            vow_bump:     10,              // In whole Dai units
-            vow_hump:     500,             // In whole Dai units
-            cat_box:      10 * THOUSAND,   // In whole Dai units
-            ilk_count:    17               // Num expected in system
+            dsr_rate:              0,               // In basis points
+            vat_Line:              1227 * MILLION,  // In whole Dai units
+            pause_delay:           60,              // In seconds
+            vow_wait:              3600,            // In seconds
+            vow_dump:              2,               // In whole Dai units
+            vow_sump:              50,              // In whole Dai units
+            vow_bump:              10,              // In whole Dai units
+            vow_hump:              500,             // In whole Dai units
+            cat_box:               10 * THOUSAND,   // In whole Dai units
+            ilk_count:             17,              // Num expected in system
+            osm_mom_authority:     address(0),      // OsmMom authority
```

Comment:
Is this something that we want to actually check every week? I'm not really sure how was checked in spells-mainnet, but maybe it could be just checked apart from the regular framework.

Reply:
I was thinking we would just remove this check from the struct next week.

Reply:
Ok sounds good.